odd_functions/intels/fn_addInteractionLocal.sqf

```diff
@@ -20,6 +20,12 @@ params ["_unit"];
         private _src = nil;
         if ((side (_this select 0)) == opfor) then {_src = "OPFOR";};
         if ((side (_this select 0)) == civilian) then {_src = "CIVIL";};
+        if ((side (_this select 0)) == blufor) then {_src = "BLUFOR";};
+
+        // Other sides from https://community.bistudio.com/wiki/Side
+        if ((side (_this select 0)) == resistance) then {_src = "OPFOR";};
```

Comment:
As we don't manage multi sides operations for now, I've put them as `OPFOR`. We may want to edit that to a new string when the civilians will be able to attack us :)

Reply:
if civilan can attack, they are no longer civilan, but resistance ...

Reply:
```suggestion
        if ((side (_this select 0)) == resistance) then {_src = "RESISTANCE";};
```